packages/http-server/src/server.ts

```diff
@@ -49,13 +49,86 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
 
   const prism = createInstance(mergedConfig, components);
 
+  const replyHandler: fastify.RequestHandler<IncomingMessage, ServerResponse> = async (request, reply) => {
+    const {
+      req: { method, url },
+      body,
+      headers,
+      query,
+    } = request;
+
+    const input = {
+      method: (method ? method.toLowerCase() : 'get') as HttpMethod,
+      url: {
+        path: (url || '/').split('?')[0],
+        query,
+        baseUrl: query.__server,
+      },
+      headers,
+      body,
+    };
+
+    request.log.info({ input }, 'Request received');
+    try {
```

Comment:
if we get rid of `throw new Error('Unable to find any decent response for the current request.');` and `throw new Error(`Cannot find serializer for ${contentType}`);` then, possibly, this `try/catch` could be turned into an Either

Reply:
That's a good point, however it's out of the scope of this PR to use fp-ts in the http-server — because we do not really know yet what's going to happen to this package in general. We'll get up a plan the day we'll have clear ideas.